tpu-client-next/src/leader_updater.rs

```diff
@@ -35,7 +35,7 @@ pub trait LeaderUpdater: Send {
     /// If the current leader estimation is incorrect and transactions are sent to
     /// only one estimated leader, there is a risk of losing all the transactions,
     /// depending on the forwarding policy.
-    fn next_leaders(&self, lookahead_slots: u64) -> Vec<SocketAddr>;
+    fn next_leaders(&mut self, lookahead_slots: u64) -> Vec<SocketAddr>;
```

Comment:
why this change?

Reply:
Because to implement this trait in STS I need to able to modify self.